db/queries/user/__init__.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+
+def is_user_registered(cursor,email: str) -> bool:
+    query = """
+        SELECT EXISTS(
+            SELECT 
+                1 
```

Comment:
The query within the EXISTS(...) is not checked. 1 is just a unit used, it could be * and it wouldn't really matter performance wise. In some you can even write EXISTS(SELECT 1/0 ...) and the result is the same - without any (division by zero) error, which proves that the expression there is not even evaluated.